modules/mailer/mail.go

```diff
@@ -52,7 +51,7 @@ func SendUserMail(c *macaron.Context, u *models.User, tpl base.TplName, code, su
 		return
 	}
 
-	msg := NewMessage([]string{u.Email}, subject, body)
+	msg := NewMessage([]string{u.Email}, subject, body, "")
```

Comment:
Looks like you're always passing empty string as `bodyPlain`, then why do you need this?